config/manager/controller_manager_config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+kind: ControllerManagerConfig
+health:
+  healthProbeBindAddress: :8081
+metrics:
+  bindAddress: 127.0.0.1:8080
+webhook:
+  port: 9443
+leaderElection:
+  leaderElect: true
```

Comment:
does this mean we now have more containers?

Reply:
no, it doesn't do it, but I left him because this [kustomization.yaml](https://github.com/IBM/ibm-block-csi-operator/blob/CSI-3123/Uograde_operator_sdk_to_1.9.0/config/manager/kustomization.yaml) file is using it

Reply:
1. do we even need the kustomization.yaml?
2. is this file mandatory for kustomization.yaml?
3. does this file have to (falsely) enable leader election?

Reply:
removed it